src/main/resources/assets/paths/progressReport.yaml

```diff
@@ -0,0 +1,35 @@
+post:
+  summary: Create a new DAR progress report
+  description: Create a new DAR progress report
+  tags:
+    - Data Access Request
+  parameters:
+    - name: parentReferenceId
+      in: path
+      description: The referenceId of the Data Access Request
+      required: true
+      schema:
+        type: string
+  requestBody:
+    content:
+      required: true
```

Comment:
This line needs to move up to the same indentation as `content:` As it is now, this renders as:
![Screenshot 2023-06-01 at 2 48 30 PM](https://github.com/DataBiosphere/consent/assets/116679/c75b1342-9cb4-4c75-9320-4695fae7e80b)